python/cuvs_bench/cuvs_bench/config/deep-1B.yaml

```diff
@@ -0,0 +1,26 @@
+dataset:
```

Comment:
I don't think we want to couple the param sweeps dirdftly to the datasets. The whole reason for separating them is that the sweeps should generalize across all datasets (and because it's more manageable to specify the sweeps once, instead of per dataset)